scripts/createImage.sh

```diff
@@ -38,11 +39,16 @@ if [[ "$SPARK_VERSION" == "3."* ]] && ( [[ "$SCALA_BIN_VERSION" == "2.13" ]] ||
     ./build/mvn --batch-mode clean
     ./build/mvn --batch-mode package -pl examples
     ./build/mvn --batch-mode package -Pkubernetes -Pscala-$SCALA_BIN_VERSION -pl assembly
-    ./bin/docker-image-tool.sh -t "$image_tag" build
+    # including python in spark image
+    ./bin/docker-image-tool.sh -t "$image_tag" -p ./resource-managers/kubernetes/docker/src/main/dockerfiles/spark/bindings/python/Dockerfile build
+    # remove extra image created by docker-image-tool.sh
+    docker image rm spark:$image_tag
```

Comment:
This should not be deleted if `INCLUDE_PYTHON` is `false`. Can we just not delete this in this script? The caller can delete it if really needed.

And I think this will not free any disk since the `spark-py` image contains it.